### ISSUE TITLE
csv2json: Read fields that has multiple line

### ISSUE THREAD

**Issue by Mike-Van** [mrodrig/json-2-csv#110]
Hello, im trying to read from csv file the fields that has multiple line. But it somehow doesnt read correctly. Here's my sample csv data:

```
2,COVET COFFEE CAMBODIA,,,,FOOD & BEVERAGE,Mr. Phalla ,General Manager,covetcoffeecambodia@gmail.com,+855 70 888 242/ 98 506 171 (sakirin),,Yes,,,,1,,,,,S,Rachana,Meeting ,Mr.Sak kirin,2/15/19,,,,"Called 7-Feb-19 He busy this week need to call again to confirm meeting on15-Feb-19
Meeting with Mr.Sak Kirin 15-Feb-19 He will report to owner",https://www.facebook.com/pg/covetcoffee/about/?ref=page_internal
```
the multiple line field is 
**"Called 7-Feb-19 He busy this week need to call again to confirm meeting on15-Feb-19
Meeting with Mr.Sak Kirin 15-Feb-19 He will report to owner"**
i try split the value by split('\r') or split('\n') but still doesnt give me the correct array.
`let notes = typeof val !== 'undefined' ? val.split('\r') : []`
it gives me this 2 seperate objects: 
```
{
        "\ufeffNo": 2,
        "Company Name": "COVET COFFEE CAMBODIA",
        "Group": "",
        "Priority": "",
        "Survey": "",
        "Sector": "FOOD & BEVERAGE",
        "Name": "Mr. Phalla ",
        "JOB TITLE": "General Manager",
        "Email": "covetcoffeecambodia@gmail.com",
        "Mobile": "+855 70 888 242/ 98 506 171 (sakirin)",
        "Email Presentation": "",
        "Presentation": "Yes",
        "Beta Sent": "",
        "Beta Participation": "",
        "Beta Signed": "",
        "# of outlets": 1,
        "Delivery Service": "",
        "# delivery points": "",
        "E-commerce": "",
        "Turnover ($m)": "",
        "Size": "S",
        "Incharge Name": "Rachana",
        "Activity ": "Meeting ",
        "Contact Person ": "Mr.Sak kirin",
        "Last Meeting ": "2/15/19",
        "Address": "",
        "Address ( District)": "",
        "Address (City )": "",
        "Notes": [
            "\"Called 7-Feb-19 He busy this week need to call again to confirm meeting on15-Feb-19",
            ""
        ],
        "Website": ""
    },
    {
        "\ufeffNo": "Meeting with Mr.Sak Kirin 15-Feb-19 He will report to owner\"",
        "Company Name": "https://www.facebook.com/pg/covetcoffee/about/?ref=page_internal\r",
        "Notes": false,
        "Website": ""
    },
```
the `Notes` object should be: 
{
```
  "Notes": [
     "Called 7-Feb-19 He busy this week need to call again to confirm meeting on15-Feb-19",
     "Meeting with Mr.Sak Kirin 15-Feb-19 He will report to owner"
  ]
}
```

**Comment by mrodrig** [comment]
Hi @Mike-Van. Thanks for reporting that issue! At initial glance, it appears that this may be a known issue (#95). Hopefully I'll have a chance this weekend to take a look at it. I'll keep you posted on the progress.

Thanks again!

**Comment by mrodrig** [comment]
Since this is the same issue as reported in # 95, I'm going to close this out. This is next up on my list of things to tackle and hopefully I'll be able to make some progress on it this weekend.

**Comment by mrodrig** [comment]
Hi @Mike-Van!
Just wanted to post back and let you know that the fix for the issue that you reported has been merged into `master` and is now released in `3.5.0` on NPM.

Please feel free to let me know if you run into any other issues.

Thanks!
